src/planner/planner.c

```diff
@@ -406,6 +406,20 @@ preprocess_query(Node *node, PreprocessQueryContext *context)
 
 					if (ht)
 					{
+						/* SELECT 1 FROM ONLY "foo"."bar" x WHERE "time" OPERATOR(pg_catalog.=) $1
```

Comment:
What's this query and why does it need special handling? Let's add some explanations to the comment.

Reply:
done